benchmarks/dbscan/dbscan.cpp

```diff
@@ -90,6 +91,13 @@ int main(int argc, char *argv[])
     return 1;
   }
 
+  if (std::set<std::string>{"fdbscan", "fdbscan-densebox"}.count(
+          params.implementation) == 0)
+  {
+    std::cerr << "Backend must be \"fdbscan\" or \"fdbscan-densebox\"\n";
+    return 2;
+  }
```

Comment:
This looks like a duplicate of the lines below and you are only changing "Implementation" to "Backend"?!